front/app/components/ProjectCard/index.tsx

```diff
@@ -529,36 +526,31 @@ const ProjectCard = memo<InputProps>(
           </ContentHeaderLabel>
         );
       } else if (timeRemaining) {
-        const totalDays = timeRemaining
-          ? moment.duration(moment(endAt).diff(moment(startAt))).asDays()
-          : null;
-        const pastDays = timeRemaining
-          ? moment.duration(moment(moment()).diff(moment(startAt))).asDays()
-          : null;
+        const totalDays = moment
```

Comment:
All these timeRemaining checks are redundant since we already check it on line 528.